packages/main/src/Popover.ts

```diff
@@ -222,7 +222,9 @@ class Popover extends Popup {
 	}
 
 	/**
-	 * Defines the ID or DOM Reference of the element that the popover is shown at
+	 * Defines the ID or DOM Reference of the element that the popover is shown at.
```

Comment:
similar:
Defines the ID or DOM Reference of the element at which the popover is shown.
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the popover.